app/services/password_crypt.rb

```diff
@@ -9,4 +9,8 @@ def self.passw_crypt(passwd, hash_alg = 'MD5')
   def self.grub2_passw_crypt(passw)
     self.passw_crypt(passw, 'MD5')
   end
+
+  def self.nix?
```

Comment:
Can we do more transparent method name? I would suggest `crypt_unix_compatible` or if we want to be precise then `crypt_mcf_compatible` (Modular Crypt Format defacto-standard: http://pythonhosted.org/passlib/modular_crypt_format.html). Definitely not "*nix" because MacOS is also unix-like OS.